packages/account-v2/src/components/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ComponentProps, HTMLAttributes, PropsWithChildren, ReactElement } from 'react';
+import { twMerge } from 'tailwind-merge';
+import { Text } from '@deriv-com/ui';
+
+type TTimelineItemProps = HTMLAttributes<HTMLDivElement> & { itemTitle: React.ReactNode };
+
+type TTimelineProps = HTMLAttributes<HTMLOListElement> & {
+    children: ReactElement<TTimelineItemProps>[];
+    lineHeight?: ComponentProps<typeof Text>['lineHeight'];
+};
+
+const TimelineItem = ({ children }: PropsWithChildren<TTimelineItemProps>) => {
+    return <div>{children}</div>;
+};
+
+const Marker = ({ label }: { label: number }) => {
+    return (
+        <div className='border absolute w-24 h-24 pl-px border-solid-red-0 rounded-full bg-solid-red-0 leading-[23.5px] mr-8 text-center -left-12'>
+            <Text className='relative text-white align-middle text-[16px] leading-normal' size='md' weight='bold'>
+                {label}
+            </Text>
+        </div>
+    );
+};
+/**
+ * @deprecated TODO: Replace this component with the one from @deriv-com/ui is implemented.
+ */
+export const Timeline = ({ children, className, lineHeight }: TTimelineProps) => {
+    if (!Array.isArray(children)) return null;
+    return (
+        <ol className={twMerge('ml-[12px]', className)}>
```

Comment:
U can use `ml-12`

Reply:
Yeah, I realised this @likhith-deriv, was refactoring it